src/transformers/inject-dependencies.js

```diff
@@ -0,0 +1,79 @@
+const { SourceMapGenerator } = require('source-map')
+
+const { listDependencies, listGlobals } = require('../utils')
+
+const generateSourceMaps = (
+  file = 'snippet.js',
+  code,
+  position,
+  dependencies,
+  globals
+) => {
+  const source = process.env.NODE_ENV === 'dev' ? `/${file}` : file
```

Comment:
I was doing this because otherwise the file path would be relative to the current directory and it would fail on different machines. That's what happening to the tests right now.